app/gui/view/src/component_browser.rs

```diff
@@ -112,7 +112,11 @@ impl component::Frp<Model> for Frp {
                 &size,
                 &gap,
                 |list_size, list_x, size, gap| {
-                    Vector2(*list_x - list_size.x / 6.0, - size.y / 2.0 - gap - NODE_HEIGHT / 2.0)
+                    let half_width = list_size.x / 2.0;
+                    let left_x = list_x - half_width;
+                    let bottom_y = -size.y / 2.0;
+                    let half_node_height = NODE_HEIGHT / 2.0;
+                    Vector2(left_x, bottom_y - gap - half_node_height)
```

Comment:
Please refactor that to a well-named function telling what these things do (they align the searcher to the left side, right?). It's hard to understand that by looking at these expressions alone. If refactorign would be hard, add comment of what this code does.

Reply:
> they align the searcher to the left side, right?

The other way around, the expression_input_position is aligned to the left side of the component browser. Refactored (also removed one unnecessary input variable).